guides/release/pages.yml

```diff
@@ -89,28 +89,28 @@
 - title: "Routing"
   url: "routing"
   pages:
-    - title: Introduction
-      url: index
-    - title: Defining Your Routes
-      url: defining-your-routes
-    - title: Linking Between Routes
-      url: linking-between-routes
-    - title: Specifying a Route's Model
-      url: specifying-a-routes-model
-    - title: Rendering a Template
-      url: rendering-a-template
-    - title: Redirecting
-      url: redirection
-    - title: Preventing and Retrying Transitions
-      url: preventing-and-retrying-transitions
+    - title: "Introduction"
```

Comment:
Just FYI this isn't necessary, it all gets compiled down to JSON anyway in the build step 👍